pkg/hive/hive.go

```diff
@@ -55,6 +55,7 @@ type Hive struct {
 	flags                     *pflag.FlagSet
 	viper                     *viper.Viper
 	lifecycle                 *DefaultLifecycle
+	healthStatus              *cell.HealthProvider
```

Comment:
nit: naming is this really a health status or the provider

Reply:
Going to remove this from this struct, we can just provide it to the container directly since Hive doesn't actually need a reference to it.